### PR TITLE
Y axis zoom widget

### PR DESCRIPTION
**Pull request by robinr**
Please find the implementation of Y-axis zoom widget it contains the followings:-
a) RadioButtonWidget .....  this provides the option selection component.
b) InputWidget             ...... this provides the option to key in Max and Min values.
c) ButtonWidget           ...... this provides the option to "Apply" or "Cancel"

Let review and let me know your comments, please

### CODE REVIEW

**Review by eli88fine — COMMENTED** [review]
looks good, just one minor comment

**Comment by jperson** [comment]
LGTM, once you push the change Eli asked for I'll approve

File: components/playback/controls/YAxisControlsSettings.vue
Comment (robinr) [reply components/playback/controls/YAxisControlsSettings.vue:82]: Old code   
 ```
    this.btnnames_y_axis_widget = ["Apply", "Cancel"];
    this.enablelist_y_axis_widget = [true, true];
    this.visiblecolor_y_axis_widget = "#B7B7B7";
    this.hidecolor_y_axis_widget = "#3F3F3F";
    this.hovercolors_y_axis_widget = ["#FFFFFF", "#FFFFFF"];
```
Modified code
    ```
this.btn_names_y_axis_widget = ["Apply", "Cancel"];
    this.enable_list_y_axis_widget = [true, true];
    this.visible_color_y_axis_widget = "#B7B7B7";
    this.hide_color_y_axis_widget = "#3F3F3F";
    this.hover_colors_y_axis_widget = ["#FFFFFF", "#FFFFFF"];
```

Modified

**Review by jperson — APPROVED** [review]

**Pull request merged**
